universal_notifications/backends/emails.py

```diff
@@ -31,7 +32,7 @@ def send_email(template, to, subject, variables={}, fail_silently=False, cms=Fal
     replace_variables['protocol'] = protocol
     domain = variables['site'].domain
     replace_variables['domain'] = domain
-    for key, value in replace_variables.iteritems():
+    for key, value in (replace_variables.iteritems() if six.PY2 else replace_variables.items()):
```

Comment:
What is wrong with six? There is nice function: `six.iteritems`.
Still three years is a lot and sometimes `.items()` is very not optimal.